platform-configs/protectli-vp2420.robot

```diff
@@ -42,6 +42,9 @@ ${USB_SHORT_NAME}=                                  ${EMPTY}
 ${ME_INTERFACE}=                                    Intel Corporation Device 4b70 (rev 11)
 ${INITIAL_FAN_RPM}=                                 ${EMPTY}
 ${ACCEPTED_%_NEAR_INITIAL_RPM}=                     ${EMPTY}
+${AUTO_BOOT_TIME_OUT_DEFAULT_VALUE}=                6
+${DASHARO_ENTER_PROMPT}=                            <Enter>=Select Entry
```

Comment:
@Psotas I really do not see why need these to be customizable. Can you give 2 platforms that have these strings set differently? We use the same payload everywhere.

You open an issue on problem with variables: https://github.com/Dasharo/dasharo-issues/issues/519 that are constantly missing in platform configs, while you introduce here another set of new variables, and to not add them to all configs, rendering new instances of this problem.

Reply:
@Psotas I can see you are keep adding these new variables to configs. But I am still not convinced on the necessity of their existence. Can you please respond to the above first?

Reply:
I misunderstood what the tests I was based on were waiting for (reset-to-defaults.robot).
Not every menu (Boot Mainteance Manager does not have one) contains `ESC to exit`.
I'm already removing unnecessary variables.

![image](https://github.com/Dasharo/open-source-firmware-validation/assets/14311655/91378e13-349d-40ac-b896-42b9a065d49a)